apps/web/src/styles/globals.css

```diff
@@ -90,4 +87,8 @@ html {
   bottom: 0px;
   z-index: -1;
   transform: rotate(-1deg);
-}
\ No newline at end of file
+}
+
+button:disabled {
+  cursor: not-allowed;
+}
```

Comment:
I don't think we should have global rules like this. You could add this to the Button component like this


```tsx
   <Component
      {...props}
      className={twMerge(
        "rounded-sm bg-blue-300 px-3 py-0.5 text-gray-800 transition-colors duration-200 ease-in-out hover:bg-blue-400 disabled:cursor-not-allowed disabled:opacity-70 disabled:hover:bg-blue-300",
        className
      )}
    >
      {children}
    </Component>
  ``